### DISCUSSION TITLE
Change the position of annotation

### DISCUSSION THREAD

**Discussion by ngvananh2508** [mwaskom/seaborn#3396] [Q&A]
when I ran this code:

colors = np.where(gsea_results["score"].head(20) >= 0, "positive", "negative")

plot = (so.Plot(data = (gsea_results.head(20).assign(**{"-log10(pval)": lambda x: -np.log10(x["pval"])})), x="-log10(pval)", y = "source", color = colors).add(so.Bar()))
plot.show()

it appeared this graph: 
![image](https://github.com/mwaskom/seaborn/assets/130348585/6be9bcae-87de-4206-90a8-e5eb02aad753)
I tried to move with move_legend method but it only works with seaborn grid, I tried but i could not change seaborn object to seaborn grid.
Can anyone help me with this problem?

Thank you very much!

**Comment by mwaskom** [comment]
Please provide a reproducible example so that it is possible to help, thanks.

**Reply by ngvananh2508** [reply]
I solved it by applying thuiop's method. Thank you very much for assisting me!

**Comment by thuiop** [comment]
You may be interested in seeing my answer here #3247 and there #2994 for moving the legend around.

**Reply by ngvananh2508** [reply]
Thank you so much! And I think you should put the import order and the alias in your code for easy to understand.
